pkg/types/types.go

```diff
@@ -17,10 +17,33 @@ package types
 import (
 	"encoding/json"
 	"fmt"
+	"os"
 )
 
 type EventType string
 
+var node string
+
+func init() {
+	node = os.Getenv("NODE_NAME")
```

Comment:
`$NODE_NAME` is defined in the pod spec, but this package `pkg/types` is used outside of the gadget pod (e.g. in the `kubectl-gadget` CLI) so I feel uneasy about using environment variables there.

What about
```
func Init(n string) {
    node = n
}
```
And getting the gadget-tracer-manager to call `Init()`?

Reply:
Thanks for catching this up. I'll handle this comment on https://github.com/kinvolk/inspektor-gadget/pull/774 as this change actually belongs to that PR and not here.